content/en/dashboards/guide/using_vega_lite_in_wildcard_widgets.md

```diff
@@ -0,0 +1,185 @@
+   },
+)
+```
+
+ {{< whatsnext desc="Additional resources:" >}}
+    {{< nextlink href="/metrics/units/#unit-list" >}}Full list of Datadog units{{< /nextlink >}}
+    {{< nextlink href="https://vega.github.io/vega-lite/docs/format.html" >}}Vega-Lite format customization{{< /nextlink >}}
+    {{< nextlink href="https://vega.github.io/vega/docs/expressions/" >}}Vega Expression Language for writing basic formulas{{< /nextlink >}}
+{{< /whatsnext >}}
+
+
+## Responsive sizing
+
+Widgets typically use responsive sizing by default, adjusting automatically to fit the available space. However, you have the option to set a fixed height for each data element, particularly if you want to enable scrolling within a bar chart. Similar to customizing colors, customizing sizing disables automatic responsive sizing.
+
+For example, you can use the following configuration to specify a height increment for each element:
+
+{{% collapse-content title="Example Vega-Lite Spec with custom height" level="h4" %}}
+{{< highlight json "hl_lines=3" >}}
+{
+    "width": 120,
+    "height": 120,
+    "data": {"url": "data/cars.json"},
+    "mark": "bar",
+    "encoding": {
+        "x": {
+            "field": "Name",
+            "scale": {"round": false}
+        },
+        "y": {"aggregate": "count"}
+    }
+}
+{{< /highlight >}}
+
+
+{{% /collapse-content %}} 
+
+## Referencing Datadog Data in Vega-Lite
+
+In Datadog, each "request" or query corresponds to a Vega [named data source][4]. The numbering for these sources starts at one. This means if your widget makes multiple requests, it generates corresponding datasets named `table1`, `table2`, and so forth.
+
+{{< img src="/dashboards/guide/using_vega_lite_in_wildcard_widgets/wildcard_multiple_requests.png" alt="Example wildcard widget with multiple requests" style="width:100%;" >}}
+
+Whenever possible, Datadog widgets preserve tag names from your request's "group by" field. For formula and function requests, such as Scalar or Timeseries, "Formula Aliases" are used as field names. For an example, see the [Wildcard widget][5] documentation.
+
+### Additional Field Information
+
+- Timeseries requests include a `_time` field for timestamps in milliseconds.  
+- Histogram request rows consist of three fields: `start`, `end`, and `count`.  
+- List request responses vary by data source. Use the [DataPreview][6] to determine available fields.
+
+### Field names with special characters
+
+Special considerations apply to field names that contain non-alphanumeric characters. Datadog Metrics tags [prohibit most non-alphanumeric characters][7]. However, not all products have this constraint and they allow characters in attribute names that may have dual meanings in Vega-Lite. These characters include square brackets `[]` and periods `.` used to access nested properties in object-shaped data, but need to be escaped because the backend flattens the data before returning it to you for /scalar and /timeseries data. 
```

Comment:
```suggestion
Special considerations apply to field names that contain non-alphanumeric characters. Datadog Metrics tags [prohibit most non-alphanumeric characters][7]. However, not all products have this constraint and they allow characters in attribute names that may have dual meanings in Vega-Lite. These characters include square brackets `[]` and periods `.` which are used to access nested properties in object-shaped data. They need to be escaped because the backend flattens the data before returning it to you for /scalar and /timeseries data. 
```
I thought it looked weird reading the `.` so added a bit for just readability and then separated this to two sentences.